src/Routes.js

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 import Main from './pages/Main/Main';
+import SellerUpload from './pages/SellerUpload/SellerUpload';
 
 class Routes extends React.Component {
   render() {
     return (
       <Router>
         <Switch>
+          <Route exact path="/SellerUpload" component={SellerUpload} />
```

Comment:
- 라우트 경로는 일반적으로 소문자로 작성하시고, 띄어쓰기 부분은 `-` 로 처리합니다!
```suggestion
          <Route exact path="/seller-upload" component={SellerUpload} />
```

Reply:
확인했습니다. 감사합니다!